inception/inception-ui-core/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/core/page/ApplicationPageBase.html

```diff
@@ -18,10 +18,10 @@
 -->
 <html xmlns:wicket="http://wicket.apache.org">
 <head>
-  <meta charset="utf-8"></meta>
```

Comment:
Have you tried building and running INCEpTION with this change? This is not valid XML - I think the Apache Wicket framework we use to render pages may not like it.

Reply:
It is valid HTML 5, so the framework should handle it. See https://cwiki.apache.org/confluence/display/WICKET/Wicket%27s+XHTML+tags#Wicket'sXHTMLtags-Elementwicket:header-items for examples.

Reply:
But I did not try building and running INCEpTION up to now (only installed Docker image).

Reply:
I'll try. From my memory, I tried this ages ago and it didn't work - maybe I mis-remember or maybe it was fixed since then. We'll see.

Reply:
Look good - no crashing and burning ;)